R/sample.R

```diff
@@ -202,6 +220,15 @@ combine_chains <- function(res) {
   details <- lapply(res, "[[", "details")
   details <- if (all(vlapply(details, is.null))) NULL else details
 
+  n_pars <- ncol(pars)
+  initial <- vapply(res, "[[", numeric(n_pars), "initial")
```

Comment:
This is to wrangle the initial values from each chain into a single matrix?

Reply:
yeah. vapply is "map over a structure and return a 'something' which we'll collect into a structure with one extra dimension". So here numeric(2) would mean we get a 2 x n_chains matrix (or the transpose - I can never remember)